src/schema/basic_classes.yaml

```diff
@@ -49,6 +49,19 @@ classes:
         multivalued: true
         inlined_as_list: true
         inlined: true
+      is_obsolete:
```

Comment:
thanks for adding these useful attributes. I would prefer that all nmdc-schema classes use slots and not attributes, but you may have a good reason for this that we just haven't discussed yet.

Reply:
sure, can do. 
I think the slot name "definition", for example, is pretty broad but its description is narrow and the mismatch there led me to make it an attribute instead of a reusable slot.  In addition, I'm not sure we want folks to apply `is_obsolete` or `is_root` to other classes?  But, for consistency, I've made them into slots.